community/socklog/socklog.setup

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+NORMAL="\033[1;0m"
+STRONG="\033[1;1m"
+RED="\033[1;31m"
+GREEN="\033[1;32m"
+runscript=/etc/sv/socklog-unix/run
+
+print_stop() {
+	local prompt="${STRONG}$1 ${RED}$2${NORMAL}"
+	printf "${prompt} %s"
+}
+
+print_start() {
+	local prompt="${STRONG}$1 ${GREEN}$2${NORMAL}"
+	printf "${prompt} %s"
+}
+
+print_stop "\nSTOP BusyBox Syslog & remove it from the Boot Runlevel ?" "[ Enter or Y to remove ]:"; read ans
+if [ -z "$ans" ] || [ "$ans" = "y" ] || [ "$ans" = "Y" ]; then
+	echo; rc-update del syslog boot; rc-service syslog stop; rc-service klogd stop
+fi
+
+print_start "\nSTART & enable socklog with runit ?" "[ Enter or Y to add ]:"; read ans
+if [ -z "$ans" ] || [ "$ans" = "y" ] || [ "$ans" = "Y" ]; then
+	if [ -f $runscript ] && [ ! -x $runscript ]; then
+		echo -e "\nenabling: $runscript"
```

Comment:
`echo -e` is very problematic, it’s not well standardized. It’s better to use `printf` instead, e.g. `printf '\nenabling: %s' "$runscript"`.